src/main/resources/static/css/style.css

```diff
@@ -2944,32 +2944,23 @@ input:checked+.form-slider:before {
 	display: none;
 }
 
-.animate
-{
+.animate {
 	transition: all 0.1s;
-	-webkit-transition: all 0.1s;
 }
 
-.action-button
-{
-	position: relative;
+.action-button {
 	padding: 5px 10px;
 	margin: 0px 5px 5px 0px;
-	float: right;
 	border-radius: 5px;
```

Comment:
Set this `border-radius` property to 0px, we don't need round edges (see main page's buttons).